### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by philduffy** [lloeki/svg-preview#5]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11)
Object.activate (/Users/philipduffy/.atom/packages/svg-preview/lib/svg-preview.coffee:11:9)
```

**Comment by dotnetCarpenter** [comment]
plus *Use the `atom-workspace` tag instead of the `workspace` class.*
courtesy of Deprecation Cop

**Comment by lloeki** [comment]
Fixed in d44bd833a31b055e6efe5ac5c69898207f649479
